src/dex.hs

```diff
@@ -170,13 +176,14 @@ optionList opts = eitherReader \s -> case lookup s opts of
 
 parseEvalOpts :: Parser EvalConfig
 parseEvalOpts = EvalConfig
-  <$> (option
+  <$> option
          (optionList [ ("LLVM", LLVM)
                      , ("LLVM-CUDA", LLVMCUDA)
                      , ("LLVM-MC", LLVMMC)
-                     , ("interp", Interp)])
-         (long "backend" <> value LLVM <> help "Backend (LLVM(default)|LLVM-CUDA|interp)"))
-  <*> (optional $ strOption $ long "logto"
+                     , ("interpret", Interpret)])
```

Comment:
How about "interpreted" or "interpreter"? "backend" sounds like a query for a noun, not a verb

Reply:
Renamed to `--backend interpreter`, thanks. Definitely agree that a noun is ideal, wasn't sure about the comfort level for using longer names.

"Swift API Design Guidelines"-level of descriptive naming [("clarity is more important than brevity")](https://swift.org/documentation/api-design-guidelines/#clarity-over-brevity) certainly seems too much, for Haskell and Dex.

Reply:
Another random renaming change that might help clarity is consistently renaming `Tab` → `Table` in types (e.g. `UTabCon` → `UTableCon`). Any thoughts here?

Reply:
Those things are internal, so I don't think that really matters. I don't think that `Tab` allows any confusion, so let's avoid changing things too much unless there's a good reason for that.